src/test/__tests__/ReactTestUtils-test.js

```diff
@@ -19,286 +22,290 @@ var ReactTestUtils;
 
 describe('ReactTestUtils', () => {
   beforeEach(() => {
+    createRenderer = require('ReactShallowRenderer').createRenderer;
     PropTypes = require('prop-types');
     React = require('react');
     ReactDOM = require('react-dom');
     ReactDOMServer = require('react-dom/server');
     ReactTestUtils = require('ReactTestUtils');
   });
 
-  it('should have shallow rendering', () => {
-    class SomeComponent extends React.Component {
-      render() {
+  // Shallow renderer only implemented for Fiber in 16+
+  if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
I'm a bit confused by this comment. Can we branch on require instead? Otherwise it looks like we stopped testing shallow renderer in Stack at all. However the Stack version is what we're currently running in www.